tokens.go

```diff
@@ -510,3 +512,12 @@ func NatsConnectionHelpers(token string, collective string, seedFile string, log
 
 	return inbox, jwth, sigh, nil
 }
+
+// IsEncodedEd25519Key determines if b holds valid characters for a hex encoded public key or seed
+func IsEncodedEd25519Key(b []byte) bool {
+	if len(b) != 64 {
+		return false
+	}
+
+	return hexEncodedMatcher.Match(b)
```

Comment:
these things are just bytes with no form or any way to detect if its actually a key - so this match should exclude anything generally found in paths in file names and such.